openapi.yaml

```diff
@@ -1878,33 +1878,32 @@ paths:
     post:
       summary: Validate a user-defined process (graph)
       operationId: validate-custom-process
-      description: >-
+      description: |-
         Validates a user-defined process without executing it. A user-defined process is
         considered valid unless the `errors` array in the response contains at
         least one error.
 
-
         Checks whether the user-defined process is schematically correct and the
```

Comment:
user-defined process -> process graph ?

Reply:
user-defined process includes the process graph, but validators may also validate the metadata.

Reply:
Clarified it a bit more.

Reply:
To me,  UDP means a process graph with a name (because the `process_graph_id` it is required in the UDP life cycle), while `/validate` also covers "anonymous" process graphs (the ones you pass as batch jobs or to `/execute`).

maybe write "process graph or user-defined process" ?

Reply:
See https://github.com/Open-EO/openeo-api/pull/316/commits/8d7aaf82ac04b3fefbd766b0f3ea51f8af58a348 - I think my changes and your comment overlapped.